### DISCUSSION TITLE
GitHub Actions is unavailable

### DISCUSSION THREAD

**Discussion by Prayasvadlakonda** [community/community#126037] [Actions] [answered]
### Select Topic Area

Question

Hello Everyone,

I tried to use GitHub actions on my account and it says `GitHub Actions workflows can't be executed on this repository.
GitHub Actions is not available for private repositories on legacy per-repository billing plans. Please visit your billing settings to upgrade your account.` I am currently on the Gold plan and would like to understand how to get Github actions in my account! Thank you
<img width="1284" alt="Screenshot 2024-05-27 at 12 05 36 PM" src="https://github.com/community/community/assets/84115263/8796d3b8-411c-4557-8205-82bd7948cf01">

**Comment by Abhishek22K** [comment, marked as answer]
The Gold plan you are on is considered a legacy plan by GitHub, and it does not include GitHub Actions for private repositories. To enable GitHub Actions for your private repositories, you'll need to switch to one of the current GitHub pricing plans that support this feature.

Steps to Enable GitHub Actions
Visit Your Billing Settings:

Go to your [GitHub billing settings](https://github.com/settings/billing).
Upgrade Your Plan:

From the billing settings page, you can view the available plans. The current plans are:
Free: Includes GitHub Actions for public repositories and limited minutes for private repositories.
Pro: Provides additional features, including more GitHub Actions minutes for private repositories.
Team: Offers features suitable for teams and additional GitHub Actions minutes.
Enterprise: For large organizations with comprehensive features and support.
Select a Suitable Plan:

Choose a plan that meets your requirements. For individual use, the Pro plan is often suitable. If you are part of a team or organization, the Team or Enterprise plan might be better.
Confirm the Upgrade:

Follow the prompts to confirm the upgrade and update your billing information as needed.
Verify GitHub Actions:

Once you've upgraded, verify that GitHub Actions is enabled by navigating to your repository settings and ensuring that GitHub Actions is active.
Additional Information
GitHub Free: This plan allows unlimited public repositories with access to GitHub Actions but with limited free minutes for private repositories.
GitHub Pro: Provides more GitHub Actions minutes for private repositories and other advanced features.
GitHub Team and Enterprise: Suitable for organizations with advanced needs and more extensive usage of GitHub Actions.
By upgrading to a plan that includes GitHub Actions, you'll be able to run workflows on your private repositories without any issues. If you encounter any problems during the upgrade process, GitHub support is available to assist you.

**Comment by Jayasurya5454** [comment]
responce#2

**Reply by kishore2345** [reply]
Res
